pages/404/redirect.tsx

```diff
@@ -0,0 +1,32 @@
+import Head from "next/head"
```

Comment:
I can't access this page directly but I should be able to, right?

Reply:
You should be able to see it at http://localhost:8080/research/research-catalog/404/redirect

Reply:
Yeah. I think it was cached or something but now I do see it. Thanks.